styles.css

```diff
@@ -0,0 +1,905 @@
+  background-color: #FF4B37;
+  font-size: 16px;
+  text-align: center;
+  font-weight: 700;
+  border-radius: 5px;
+  color: white;
+  padding: 8px 30px;
+  text-decoration: none;
+  display: inline-block;
+}
+
+.button:hover {
+  background-color: #BEBEBA;
+  color: white;
+}
+
+.sub {
+  background-color: #17406C;
+  border: 2px solid white;
+}
+
+.buttons {
+  width: 100%;
+  padding-bottom: 50px;
+}
+
+.graph {
+  width: 65%;
+  padding-right: 25px;
+}
+
+p {
+  font-size: 18px;
+  font-weight: 300;
+  text-align: justify;
+  color: white;
+  clear: both;
+  line-height: 20px;
+  padding-top: 15px;
+}
+
+section p {
+  color: #6D6E71;
+  padding: 20px 15px 0 15px;
+}
+
+section {
+  background-color: white;
+  width: 100%;
+  padding-top: 40px;
+  padding-bottom: 40px;
+  height: auto;
+}
+
+section img {
+  display: block;
+  margin: auto;
+  padding-bottom: 40px;
+}
+
+.lefthome {
+  border-width: 1px;
+  border-color: #BEBEBA;
+  border-right-style: solid;
+  width: 33%;
+  height: 400px;
+}
+
+.centerhome {
+  width: 33%;
+  height: auto;
+}
+
+.righthome {
+  border-width: 1px;
+  border-color: #BEBEBA;
+  border-left-style: solid;
+  width: 33%;
+  height: 400px;
+}
+
+section a {
+  color: #17406C;
+}
+
+footer {
+  background-color: #222B30;
+  height: auto;
+  margin: 0 auto 0 auto;
+}
+
+footer p {
+  padding: 35px 15px 5px 15px;
+  font-size: 12px;
+  text-align: left;
+  line-height: 22px;
+}
+
+.leftfooter {
+  width: 33%;
+}
+
+.leftfooter p {
+  font-size: 14px;
+}
+
+.centerfooter {
+  width: 33.66%;
+}
+
+.centerfooter p {
+  text-align: center;
+}
+
+.rightfooter {
+  width: 33%;
+}
+
+.rightfooter p {
+  text-align: justify;
+  line-height: 15px;
+  padding-left: 35px;
+  padding-top: 10px;
+}
+
+#show-this-on-click {
+  padding-top: 10px;
+}
+
+#show-on-click {
+  padding-top: 10px;
+}
+
+.readless, .readmore, .moreinfo, .lessinfo {
+  padding-top: 10px;
+}
+
+footer a {
+  font-weight: 400;
+}
+
+
+iframe {
+  display: block;
+  margin: 20px auto 0 auto;
+  width: 90%;
+  height: auto;
+  border: 3px solid #BEBEBA;
+  border-radius: 8px;
+}
+
+iframe:hover {
+  border: 3px solid #FF4B37;
+}
+
+.footer-logo {
+  height: 90%;
+  width: auto;
+  padding: 35px 0 0 35px;
+  border: none;
+}
+
+h1 {
+  font-size: 48px;
+  font-weight: 700;
+  color: white;
+}
+
+h2 {
+  font-size: 24px;
+  color: #17406C;
+  font-weight: 600;
+  text-align: center;
+}
+
+h3 {
+  font-size: 16px;
+  color: white;
+  font-weight: 400;
+  text-align: center;
+  padding-top: 35px;
+}
+
+.hide {
+  display: none;
+}
+
+/*====================================================
+Tablet View
+====================================================*/
+
+@media (max-width: 768px) {
+
+  * {
+    -moz-box-sizing: border-box; /* Firexfox */
+    -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+    box-sizing: border-box;
+  }
+
+  .clearfix:after {
+       content: "";
+       display: table;
+       clear: both;
+  }
+
+  .align-left {
```

Comment:
You don't need to reiterate classes that have the same selector and value when writing media queries.